src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -464,7 +464,9 @@ object EntityClient {
       for {
         token <- getAuthenticationToken
         entities <- identifiers.parTraverse(identifier => entitiesForIdentifier(identifier, token))
-      } yield entities.flatten.toMap
+      } yield entities.flatten.groupBy(_._1).map { case (key, pairs) =>
```

Comment:
I'm not sure I understand why this was added; isn't `entities` already unique? It's a `Seq` of `Map[String, Seq[Entity]]` each Map only has one key-value pair in it, and each of these Maps has a different key, right?

Reply:
You're right, this isn't necessary. I've taken it out